pkg/config/types/engines.go

```diff
@@ -27,7 +31,7 @@ type Docker struct {
 	ManifestCache DockerManifestCache `yaml:"ManifestCache,omitempty"`
 }
 
-func (d Docker) Installed() bool {
+func (d Docker) IsConfigured() bool {
 	return d != Docker{}
 }
```

Comment:
This doesn't mean much. Docker can still be enabled with default and empty configuration. I appreciate what you are doing with this method, but we already have `IsInstalled()` in the engines themselves that are more accurate